@xen-orchestra/web-core/lib/components/ui/radio-button/UiRadioButton.vue

```diff
@@ -0,0 +1,185 @@
+    align-items: center;
+    justify-content: center;
+    border: 0.2rem solid var(--accent-color);
+    background-color: transparent;
+    border-radius: 0.8rem;
+    width: 1.6rem;
+    height: 1.6rem;
+    position: relative;
+    transition:
+      border-color 0.125s ease-in-out,
+      background-color 0.125s ease-in-out;
+
+    &:has(input:focus-visible) {
+      outline: none;
+      &::after {
+        position: absolute;
+        content: '';
+        inset: -0.5rem;
+        border: 0.2rem solid var(--color-info-txt-base);
+        border-radius: 0.4rem;
+      }
+    }
+    &:has(.input:disabled) {
+      cursor: not-allowed;
+
+      /*
+      TODO: To be removed or kept after a decision has been taken
+      See https://www.figma.com/design/l2O2VvzJRnOCvqxhM7d124?node-id=8-1940#1021964394
+      */
+
+      &:not(:has(.input:checked)) {
+        --accent-color: var(--color-neutral-txt-secondary);
+      }
+    }
+
+    &:has(.input:checked) {
+      background-color: var(--accent-color);
+    }
+
+    /* INPUT */
+
+    .input {
+      opacity: 0;
+      position: absolute;
+      pointer-events: none;
+    }
+
+    /* ICON */
+
+    .radio-icon {
+      font-size: 0.8rem;
+      position: absolute;
+      opacity: 0;
+      transition: opacity 0.125s ease-in-out;
+      color: var(--color-neutral-txt-primary);
+    }
+
+    .input:disabled + .radio-icon {
+      color: var(--color-neutral-txt-secondary);
+    }
+
+    .input:checked + .radio-icon {
+      opacity: 1;
+    }
+  }
+
+  /* ACCENT */
+
+  &.accent--info {
+    --accent-color: var(--color-info-item-base);
+
+    &:hover {
+      --accent-color: var(--color-info-item-hover);
+    }
+
+    &:active {
+      --accent-color: var(--color-info-item-active);
+    }
+
+    &:has(.input:disabled) {
+      --accent-color: var(--color-info-item-disabled);
+    }
+  }
+
+  &.accent--success {
+    --accent-color: var(--color-success-item-base);
+
+    &:hover {
+      --accent-color: var(--color-success-item-hover);
+    }
+
+    &:active {
+      --accent-color: var(--color-success-item-active);
+    }
+
+    &:has(.input:disabled) {
+      --accent-color: var(--color-success-item-disabled);
+    }
+  }
+
+  &.accent--warning {
+    --accent-color: var(--color-warning-item-base);
+
+    &:hover {
+      --accent-color: var(--color-warning-item-hover);
+    }
+
+    &:active {
+      --accent-color: var(--color-warning-item-active);
+    }
+
+    &:has(.input:disabled) {
+      --accent-color: var(--color-warning-item-disabled);
+    }
+  }
+
+  &.accent--danger {
+    --accent-color: var(--color-danger-item-base);
+
+    &:hover {
+      --accent-color: var(--color-danger-item-hover);
+    }
+
+    &:active {
+      --accent-color: var(--color-danger-item-active);
+    }
+
+    &:has(.input:disabled) {
+      --accent-color: var(--color-danger-item-disabled);
+    }
+
+    .radio-container .radio-icon {
+      color: var(--color-danger-txt-item);
+    }
```

Comment:
I think this selector can be simplified to:
```suggestion
    .radio-icon {
      color: var(--color-danger-txt-item);
    }
```
And it should be used in the other accent variants with the right values, since the icon color also changes for `info`, `success` and `warning`. Here is a screenshot of the design system; we can see the value is `--color-info-txt-item` for the `info` variant:
![image](https://github.com/user-attachments/assets/d9763c0f-6412-4470-9319-1c7faa16f73a)